code/index.html

```diff
@@ -2,21 +2,86 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Hearthn00b</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+
+      <div class="hamburger"> 
+
+        <div></div>
+        <div></div>
+        <div></div>
+
+      </div>
+
+      <img src="hearthnoob.png" alt="hearthnoob logo">
+
+      <nav>
+
+        <a href="#"> BIG ARTICLE</a>
+        <a href="#"> OTHER ARTICLES</a>
+
+      </nav>
+    
     </header>
 
+      <!--introduction section-->
+
+    <section class="introduction">
+
+      <h1>Great hearthstone decks for beginners!</h1>
+
+    </section>
+
+       <!--Big news section-->
     <section class="big-news">
-      <!-- The big news content -->
+
+      <article>
+        <a href="https://hsreplay.net/decks/#playerClasses=HUNTER&archetypes=355" target="_blank">
+          <img src="toxic reinforcements.png" alt="toxic reinforcements">
```

Comment:
Good that you've used the alt in the img tag but try to describe what the picture is. Here's some info on why it's important but just to keep in mind:
https://www.w3schools.com/tags/att_img_alt.asp